src/main/java/solaire/Solaire.java

```diff
@@ -0,0 +1,86 @@
+package solaire;
+
+import java.util.Scanner;
+
+import solaire.storage.Storage;
+import solaire.tasklist.TaskList;
+import solaire.ui.Ui;
+
+/**
+ * Represents the chatbot, Solaire.
+ * The chatbot can keep track of various types of tasks and mark/unmark tasks as complete.
+ */
+public class Solaire {
+    private TaskList taskList;
+    private Ui ui;
+
+    enum UserCommands {
```

Comment:
Maybe you might want to create a separate enums class UserCommands instead of what is done here? 🤔